spring-cloud-aws-starters/spring-cloud-aws-starter-metrics/pom.xml

```diff
@@ -35,6 +35,10 @@
 			<groupId>io.awspring.cloud</groupId>
 			<artifactId>spring-cloud-aws-autoconfigure</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Similar to other starters, `metrics` module should depend on:

```xml
<dependency>
	<groupId>io.awspring.cloud</groupId>
	<artifactId>spring-cloud-aws-starter</artifactId>
</dependency>
```

and the dependency to `spring-cloud-aws-autoconfigure` should be removed.

Reply:
@maciejwalkowiak , Should I raise a PR adding it but I guess adding to the starter will bring #427 back

Reply:
Just update this PR. #427 was only about `spring-cloud-aws-starters` module (not "starter") which used to be an aggregator module for all the starters, so there's no risk of bringing it back :)

Reply:
Updated